core/accounting.go

```diff
@@ -0,0 +1,72 @@
+package core
+
+import (
+	"math/big"
+	"sync"
+	"time"
+)
+
+// Use transcodeLoopTimeout as TTL for a ManifestID's credit balance
```

Comment:
I think its fine to use `transcodeLoopTimeout`, but curious what your motivation for using it was?

Reply:
Because it didn't make sense to me to keep track of a balance after the transcode loop times out. 
We're not bound to use this setting however and could do it another way if you prefer that.

Reply:
> it didn't make sense to me to keep track of a balance after the transcode loop times out.

Gotcha. Just wanted to point out that even if we use `transcodeLoopTimeout` here, the clean up loop will not necessarily run at the same time as the timeout for a transcode loop. I think the just the name `transcodeLoopTimeout` here might be a bit misleading for some readers. What if we passed the value as a parameter similar to what is done for the [pm.SenderMonitor](https://github.com/livepeer/go-livepeer/blob/master/pm/sendermonitor.go#L94)? We could even use the same top level [value](https://github.com/livepeer/go-livepeer/blob/master/cmd/livepeer/livepeer.go#L57).

Reply:
Crossed my mind to define it on the struct itself and after second thought that might be better. 

I'd like to use https://github.com/livepeer/go-livepeer/blob/a7ae0133c97ed05d77c130dcc9faf0b5392f7976/cmd/livepeer/livepeer.go#L55 though just because it's a `time.Duration` type as well. Not that it matters that much because we are converting it to int64 , but at least we're checking two `int64` casted `time.Duration `types that way.